doc/howtos/rdtraining/architecture.rst

```diff
@@ -0,0 +1,100 @@
+.. _howto/rdtraining/architecture:
+
+=====================
+Architecture Overview
+=====================
+
+Multitier application
+=====================
+
+Odoo follows a `multitier architecture`_ architecture, meaning that the presentation, the business
+logic and the data storage are separated. More specifically, it uses a three-tier architecture
+(image from Wikipedia):
+
+.. image:: architecture/media/three_tier.svg
+    :align: center
+    :alt: Three-tier architecture
+
+The presentation tier is a combination of HTML5, JavaScript and CSS. The logic tier is exclusively
+written in Python, while the data tier only supports PostgreSQL as an RDBM.
```

Comment:
The data tier includes the persistence mechanism (PostgreSQL as an RDBM) and the Odoo's ORM that encapsulates the persistence mechanisms and exposes the data

Reply:
XML files and CSV files are also a part of the data tier